packages/docs/pages/usage/search/sorting.mdx

```diff
@@ -0,0 +1,93 @@
+# Sorting
+
+To sort, Orama uses the properties defined in the `schema` to know on which properties you want to sort.
+
+```javascript copy
+const db = await create({
+  schema: {
+    title: 'string',
+    year: 'number',
+    isTop: 'boolean',
+    meta: {
+      tag: 'string',
+      rating: 'number',
+      favorite: 'boolean',
+    },
+  }
+})
+const results = await search(db, {
+  term: 'prestige',
+  sort: {
+    property: 'title', // or 'year', 'isTop'
+  }
+})
+```
+
+Orama supports sorting on 'string' and 'number'.
+
+You can also specify nested properties using the '.' notation: `'meta.tag'`, `'meta.rating'` and `'meta.favorite'`
+
+## Reverse order
+
+Orama supports the reverse order specifying the key `order`:
+
+```javascript
+
+const db = await create({
+  schema: {
+    title: 'string',
+    year: 'number',
+    isTop: 'boolean',
+    meta: {
+      tag: 'string',
+      rating: 'number',
+      favorite: 'boolean',
+    },
+  }
+})
+const results = await search(db, {
+  term: 'prestige',
+  sort: {
+    property: 'title', // or 'year', 'isTop'
```

Comment:
We don't sort booleans, so `isTop` should not appear here

Reply:
See my comment above.